### ISSUE TITLE
Unhandled error in statsd mapper

### ISSUE THREAD

**Issue by philbooth** [nature/boomcatch#60]
```
TypeError: Cannot read property 'type' of undefined
    at /usr/lib/node_modules/boomcatch/src/mappers/statsd.js:73:21
    at Array.map (native)
    at mapRestimingMetrics (/usr/lib/node_modules/boomcatch/src/mappers/statsd.js:69:17)
    at /usr/lib/node_modules/boomcatch/src/mappers/statsd.js:61:23
    at Array.forEach (native)
    at map (/usr/lib/node_modules/boomcatch/src/mappers/statsd.js:51:23)
    at send (/usr/lib/node_modules/boomcatch/src/index.js:656:22)
    at IncomingMessage.emit (events.js:92:17)
    at _stream_readable.js:943:16
    at process._tickCallback (node.js:419:13)
```

**Comment by philbooth** [comment]
Hey @roflmao, I'm trying to reproduce this using some little test pages that I added to the repo last week:

* https://github.com/nature/boomcatch/blob/master/client/restiming-get.html
* https://github.com/nature/boomcatch/blob/master/client/restiming-post.html

So far I've failed to reproduce it, although I have uncovered another issue that I need to fix: #61. Maybe this problem is something specific to your payloads?

I appreciate you might not be able to post your entire page here (though if you can, that's cool), but would it be possible to try and modify one of the above test cases so that it reproduces the bug? That would be really helpful in tracking down the cause of the problem.

There's a chance it could also be related to some difference in the versions of boomerang that we're running against. Which commit hash and build options have you run it with or, even better, can you point me at the built `.js` file you're using so I can run against that?

**Comment by philbooth** [comment]
Hey @roflmao, I think I've fixed this in version 3.0.2, just published. Let me know if it doesn't work.

Apologies for all the problems you've had. We're not actually using the resource-timing stuff at Nature so it's been sitting unused since I first wrote it. As far as I know, you're the guinea pig! :)

**Comment by roflmao** [comment]
Excellent! Works like a charm :)
